### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute (part 2)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by brittmsantos** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@IEvangelist @gewarren @jasonwhowell

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: Contribute/how-to-write-links.md [hunk 1]
```diff
@@ -68,20 +68,16 @@ A file link is used to link from one article to another within the docset.
...
-```
+`https://docs.microsoft.com/<locale>/<product-service>/[<feature-service>]/[<subfolder>]/<topic>[?view=<view-name>]`
 
 Examples:
 
-```
-https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview
-https://docs.microsoft.com/en-us/powershell/azure/overview?view=azurermps-5.1.1
-```
+`https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview
```
Comment (gewarren) [inline Contribute/how-to-write-links.md:79]: Why not use the multi-line code fencing here?

Comment (alexbuckgit) [reply Contribute/how-to-write-links.md]: Reverted :-)

File: Contribute/index.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Microsoft Docs contributor guide overview
-description: The guide describes how you can contribute to the Microsoft documentation site docs.microsoft.com.
+title: Contributor guide overview
```
Comment (JasonWHowell) [inline Contribute/index.md:2]: This impacts SEO. @jehchow do you have an opinion on the public contributor guide page?

**Review by JasonWHowell — COMMENTED** [review]
Looks fine. I don't know if we should change the name of the contributor guide.

File: Contribute/index.md (hunk 2 above)
Comment (alexbuckgit) [reply Contribute/index.md]: Reverted

**Pull request merged**